platform/android/export/export.cpp

```diff
@@ -2223,7 +2225,7 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return list;
 	}
 
-	void _update_custom_build_project(const String &output_apk_path) {
+	void _update_custom_build_project() {
 
```

Comment:
Not sure if you saw the general comment - but please remove these extra newlines at the beginning of blocks/functions